canonical-web-and-design/jaas-dashboard
# Add actions button row interactions

## Done

- Created actions button row in Entity Details App Units page.
- Passed unit values to the Actions panel.
- Updated content in the Actions panel based on the unit list.

## QA

- Contact me if you don't have access to the `actions-test` model.
- View the application and select a unit in the row and click the "Run Action" button.
- With the panel open, click another unit, the panel should update.
- Deselect all units, the panels should gracefully handle no units selected.
- Submit an action.

## Details

Fixes: https://github.com/canonical-web-and-design/juju-squad/issues/1655
Fixes: https://github.com/canonical-web-and-design/juju-squad/issues/1659

## Screenshots

![Screen Shot 2021-04-01 at 11 50 45 AM](https://user-images.githubusercontent.com/532033/113334110-8795c080-92e0-11eb-980e-de93c22599a0.png)
